chronograf.go

```diff
@@ -736,20 +736,35 @@ type OrganizationsStore interface {
 	DefaultOrganization(ctx context.Context) (*Organization, error)
 }
 
+// Config is the global application Config for parameters that can be set via
+// API, with different sections, such as Auth
+type Config struct {
+	Auth      AuthConfig        `json:"auth"`
+	LogViewer LogViewerUIConfig `json:"logViewer"`
```

Comment:
Should this be scoped by organization?

Reply:
definitely -- good catch

Reply:
@desa does this mean we should take LogViewer off the Config struct?